doc/source/calc_functions.rst

```diff
@@ -1,6 +1,6 @@
 .. vale off
 
-.. _caculator_functions:
+.. _calculator_functions:

```

Comment:
This change effects two other files that need to be updated as well:

replacements/cl_sphinx/variables.xml:    :ref:`Calculator Functions &lt;caculator_functions&gt;`
replacements/ens_globals_methods.xml:        The expression to evaluate as the new variable.  See also: :ref:`Calculator Functions &lt;caculator_functions&gt;`.